source/reference/mongosync.txt

```diff
@@ -296,6 +296,25 @@ error occurs, the ``mongosync`` log may contain the word "error' but
 ``mongosync`` is still able to complete the sync. In the case that a
 sync does not complete, ``mongosync`` writes a fatal log entry. 
 
+UUIDs
+~~~~~
+
+``mongosync`` creates new :abbr:`UUIDs (universally unique identifiers)`
+for collections on the destination cluster. There is no relationship
+between UUIDs on the source cluster and the destination cluster. If
+applications contain hard-coded UUIDs (which MongoDB does not
+recommend), you may need to update the application's UUIDs before the
+application works properly with a migrated cluster. 
```

Comment:
One last comment. Can you actually change the UUIDs? I would be surprised if this were the case.

Reply:
Checked in with Alex Komyagin who said the "The developer would need to get the new UUID's from the new cluster or update the app not to rely on UUIDs"  so I changed the phrasing accordingly